lib/restivus.coffee

```diff
@@ -307,25 +307,30 @@ class @Restivus
 
       After the user is logged out, the onLoggedOut hook is called (see Restfully.configure() for adding hook).
     ###
```

Comment:
This comment should be moved back to its original place, above the call to `@addRoute()` immediately below (for accuracy).